examples/Libevent/main.cpp

```diff
@@ -0,0 +1,6 @@
+#include <event2/event.h>
```

Comment:
Got error:
```
main.cpp:1:26: fatal error: event2/event.h: No such file or directory
```

Reply:
Weird. I thought this part worked. Then again, may be local machine setup. Will try again and update.

Reply:
If we are talking about Linux you can use Docker to reproduce issues on Travis CI, see:
* https://github.com/hunter-packages/travis-linux-docker

Reply:
Good idea. Will do.